changelog.d/127.feature

```diff
@@ -0,0 +1 @@
+Add fallback_content handling from push data.
```

Comment:
Sure, `fallback_content` is needed because we need to add `alert` in the `aps` field of payload. The handling is this: if we see this value as _true_ in the data, we're adding `alert` to the payload. It's because of this: 

> The system executes your notification content app extension only when a remote notification’s payload contains the following information:
> - The payload must include the mutable-content key with a value of 1.
> - The payload must include an alert dictionary with title, subtitle, or body information.

Detailed info can be found at https://developer.apple.com/documentation/usernotifications/modifying_content_in_newly_delivered_notifications 

It only affects iOS and does not affect Android or other pusher types.